resources/js/src/app/services/AddressService.js

```diff
@@ -10,20 +10,21 @@ const CheckoutService = require("services/CheckoutService");
  */
 export function createAddress(address, addressType, setActive)
 {
-    return ApiService.post("rest/io/customer/address?typeId=" + addressType, address).done(function(response)
-    {
-        if (setActive)
+    return ApiService.post("rest/io/customer/address?typeId=" + addressType, address, {supressNotifications: true})
```

Comment:
supressNotifications wird doch über eine config Einstellung in Ceres gesetzt oder? Überschreibst du das dann nicht hier statisch?